TRADE-MANAGER.md

```diff
@@ -0,0 +1,104 @@
+# Trade Manager
+The trade manager handles all incoming and outgoing trade offers for you! No need to invite, kick and whisper players manually anymore!
+
+## Features
+### Incoming trade offer
+When you receive a trade offer whisper, a tile with the name of the item and the price will appear at the bottom of the screen.
+
+For example, 
+``@From PlayerABC: Hi, I would like to buy your Saqawal's Nest listed for 3.7 exalted in Standard``
+
+![image](https://user-images.githubusercontent.com/25111613/86523078-e0f5bf80-be34-11ea-9a89-f07ab023e059.png)
+
+Several actions are available: 
+- Invite the buyer to your party
+- Say "My item is sold" to the buyer (Removes the offer from the overlay).
+
+  For example, ``Sorry, my Saqawal's Nest is already sold.``
+  
+- Say "I'm busy" to the buyer.
+
+  For example, ``I'm busy right now, I will send you party invite when I'm ready.``
+  
+- Say "Are you still interested in the my item?" (Usefull if you received trade offers while being AFK).
+
+  For example, ``Are you still interested in my Saqawal's Nest listed for 3.7 exalted?``
+  
+- Ignore the offer (Removes the trade offer from the overlay)
+
+![image](https://user-images.githubusercontent.com/25111613/86523224-27e4b480-be37-11ea-9e2c-ecf1199d1028.png)
+
+Once you have invited the buyer to your party, the trade offer tile will have a green indicator.
+
+![image](https://user-images.githubusercontent.com/25111613/86523308-78a8dd00-be38-11ea-9306-28ea830ca305.png)
+
+When the buyer join you hideout (or the area you're in), a "User" icon will appear at the bottom right of the trade offer tile.
+
+![image](https://user-images.githubusercontent.com/25111613/86523345-303def00-be39-11ea-9d10-75145f4e85a0.png)
+
+You'll have now, several actions available:
+- Send a trade request to the buyer (to engage the trade in-game)
+- Invite the buyer to your party (Usefull if the buyer, didn't accept your first party invite quick enough and needs another one.)
+- Kick the buyer out of your party (Usefull when the game prevent the buyer from joining your hideout with the "Failed to join instance" bug. You can then Kick the buyer and invite him again quickly)
+- Remove the offer (Removes the trade offer from the overlay **and** kick the buyer out of your party)
+
+![image](https://user-images.githubusercontent.com/25111613/86523398-d4279a80-be39-11ea-8af0-c9392eb399df.png)
+
+Once the trade is successful, the buyer will be automatically kicked out of your party and a "Thanks" whisper will be sent to him. (The trade offer is removed from the overlay aswell)
```

Comment:
> and a "Thanks" whisper will be sent to him. (The trade offer is removed from the overlay aswell)

Bugged, doesn't work. Kicked, no thank you. 
 ![Kick, no thanks :( ](https://i.imgur.com/eutTjJH.png)


Works if you disable Auto-Kick in settings. Seems it doesn't wait for the message to be sent before kicking the trader out. 

![Without auto-kick](https://i.imgur.com/SGvajvg.png)